app/(api)/_datalib/auth/createToken.mjs

```diff
@@ -0,0 +1,6 @@
+import jwt from 'jsonwebtoken';
```

Comment:
use typescript files. Mjs is only used when we need a script to run this code, but if Next.js is running the code, then use ts/tsx. Also make the expiration time an environment variable in .env. You may need to make a .env file.